.gitignore

```diff
@@ -64,3 +66,6 @@ ipch/
 CMakeLists.txt
 compile_commands.json
 /.gitmodules
+
+# .clang-format later to be removed if introduced in ES
+/.clang-format
```

Comment:
@quyykk I separated this from other ignored files so that it can be easier to find with comments

Reply:
These change to the .gitignore file have no business in this PR and should be removed.

(And you should also not force-push unless you absolutely need to overwrite your branch history. It's very disruptive. If you want to have the latest changes from upstream, use a normal merge commit instead of rebasing.)

Reply:
> These change to the .gitignore file have no business in this PR and should be removed.

It's a gitignore file... it is being updated as intended.  I'm developing with an IDE... these are IDE-related files which I'm sure you wouldn't want committed by mistake.

My IDE: https://github.com/samrocketman/endless-sky-vscode-devcontainer

---

> And you should also not force-push unless you absolutely need to overwrite your branch history. It's very disruptive. If

Agree to disagree I guess.  I can use merge commits if it makes review easier.  Though feels a bit odd to have a workflow forced on me in my development branch.  No force-pushing on public branches makes sense, though.

Reply:
Pushed new commit reverting change